test/module/cfn_json/test_cfn_json.py

```diff
@@ -64,10 +66,11 @@ def test_success_parse(self):
             filename = values.get('filename')
             failures = values.get('failures')
             template = json.load(open(filename), cls=cfnlint.cfn_json.CfnJSONDecoder)
-            cfn = Template(filename, template, ['us-east-1'])
-
+            runner = Runner(
```

Comment:
Switch to Runner which removes duplicates.  Important part of running the processor now.  We don't need duplicate alerts for something that failed across all the scenarios.